src/commands/dev/dev.js

```diff
@@ -253,7 +254,29 @@ const dev = async (options, command) => {
     )
   }
 
-  await injectEnvVariables({ env: command.netlify.cachedConfig.env, site })
+  const startNetlifyGraphWatcher = Boolean(options.graph)
+  const netlifyToken = await command.authenticate()
```

Comment:
This will open up a browser and authenticate if users are not logged in.
This is not the expected behavior for the CLI as `ntl dev` should not require it.

Reply:
we'll need the netlify bearer token for the user. What's an alternative way of getting it?

Reply:
That's the way to get it. We should only do it when running with `graph` I believe

Reply:
There’s also the API Authentication use case. I can do it only in the presence of an authlify token ID, then. Appreciate the review!